apiserver/plane/api/views/asset.py

```diff
@@ -73,3 +74,16 @@ def delete(self, request, asset_key):
             # Delete the file object
             file_asset.delete()
             return Response(status=status.HTTP_204_NO_CONTENT)
+
+class AssetsEndpoint(BaseAPIView):
+
+    permission_classes = [WorkspaceEntityPermission]
+    parser_classes = (MultiPartParser, FormParser)
+
+    def get(self, request, workspace_id, asset_key):
```

Comment:
Can we do a `slug` here instead of `workspace_id` ? @NarayanBavisetti

Reply:
Current permission layer works on `slug`

Reply:
ok sure, will change to slug